api/test/integration/env/tools/healthcheck_utils.py

```diff
@@ -92,16 +92,26 @@ def get_master_health():
     os.system("/var/ossec/bin/wazuh-control status > /tmp/daemons.txt")
     check0 = check(os.system("diff -q /tmp/output.txt /tmp/healthcheck/agent_control_check.txt"))
     check1 = check(os.system("diff -q /tmp/daemons.txt /tmp/healthcheck/daemons_check.txt"))
-    check2 = get_response(login_url, get_login_header(user, password)) is None
+    check2 = get_manager_health() is None
```

Comment:
API is not required to work on worker nodes. Distributed API requests are executed by the cluster.